src/Microsoft.AspNetCore.Mvc.ViewFeatures/Rendering/TagBuilder.cs

```diff
@@ -298,37 +298,69 @@ public void WriteTo(TextWriter writer, HtmlEncoder encoder)
                 throw new ArgumentNullException(nameof(encoder));
             }
 
-            switch (TagRenderMode)
+            WriteToHelper(this, writer, encoder, TagRenderMode);
+        }
+
+        /// <summary>
+        /// Returns the <see cref="IHtmlContent"/> body.
+        /// </summary>
+        /// <returns>The inner <see cref="IHtmlContent"/>.</returns>
+        public IHtmlContent RenderBody() => _innerHtml;
+
+        /// <summary>
+        /// Renders the start tag for the given <see cref="TagBuilder"/>.
```

Comment:
More like `Returns an <see cref="IHtmlContent" /> that renders` &hellip; Same for `<returns>` and the other added methods.